File: src/lletreRepetidesJava/Fase4.java
/**
 * Fase4 - Ejercico de LletresRepetides
 * @author Rubén Rodríguez
 * IT Academy
 * Barcelona Activa
 *
 */

package lletreRepetidesJava;

import java.util.ArrayList;
import java.util.List;

public class Fase4 {
	
	private List<Character> listNombre = new ArrayList<Character>();
	private List<Character> listCognom = new ArrayList<Character>();
	
	
	protected Fase4() {
		listNombre.add('R');
		listNombre.add('u');
		listNombre.add('b');
		listNombre.add('e');
		listNombre.add('n');
		listCognom.add('R');
		listCognom.add('o');
		listCognom.add('d');
		listCognom.add('r');
		listCognom.add('i');
		listCognom.add('g');
		listCognom.add('u');
		listCognom.add('e');
		listCognom.add('z');
	}
	
	
	protected void fase4Print () {
		
		listNombre.forEach(letra->{
			System.out.print(letra);
		});
		listCognom.forEach(letra->{
			System.out.print(letra);
		});
		
	}
	
}
	

File: bin/LletresRepetidesJavaScript/Fase4.js
/**
 * Fase4 - Ejercico de LletresRepetides en JavaScript
 * @author Rubén Rodríguez
 * IT Academy
 * Barcelona Activa
 *
 */
let listNombre4 = ['R','u','b','é','n']
let listCognom4 = ['R','o','d','r','í','g','u','e','z']
let impresFs4 = false


  
var respostaFase4 = function(){
	

	if (impresFs4){}
	else {
		impresFs4 = true
		let selectElement = document.getElementById('fase4Sortida')
		listNombre4.forEach(function(letra) {
			selectElement.innerHTML += letra+" "})
		
		listCognom4.forEach(function(letra) {
			selectElement.innerHTML += letra+" "}) 
		
	}
}

if (document.addEventListener){
    window.addEventListener('load',(function() {
    	document.getElementById('4').onclick = respostaFase4
    }),false)
} else {
    window.attachEvent('onload',(function() {
    	document.getElementById('4').onclick = respostaFase4
    }))  
}

